package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "devDependencies": {
+      "babel-eslint": "^10.1.0",
+      "eslint": "^7.32.0",
+      "eslint-config-airbnb-base": "^14.2.1",
+      "eslint-plugin-import": "^2.23.4",
+      "hint": "^6.1.4",
+      "stylelint": "^13.13.1",
+      "stylelint-config-standard": "^21.0.0",
+      "stylelint-csstree-validator": "^1.9.0",
+      "stylelint-scss": "^3.19.0"
+    }
+  }
```

Comment:
Package.json and package-lock.json are not needed for the linters of this project. Please delete those files.